src/sentry/testutils/cases.py

```diff
@@ -943,6 +944,22 @@ def assertDoesNotPass(self, rule, event=None, **kwargs):
         assert rule.passes(event, state) is False
 
 
+class DRFPermissionTestCase(TestCase):
```

Comment:
I figure I'm going to be testing a lot of new permission classes, so it'd be good to make a useful base test case